src/CameraControls/CameraControls.tsx

```diff
@@ -117,6 +118,9 @@ export const CameraControls: FC<
     const isOrbiting = mode === 'orbit';
     const setPanning = useStore(state => state.setPanning);
     const isDragging = useStore(state => state.draggingIds.length > 0);
+    const [cameraSpeed, setCameraSpeed] = useState(
```

Comment:
I would just store this in a ref, there is no need to put in a state object and trigger a refresh since this is outside react anyway.